docs/requirements.txt

```diff
@@ -1,27 +1,33 @@
-# primary
-sphinx==6.1.3
-
-# dependencies
 alabaster==0.7.13
 Babel==2.11.0
+beautifulsoup4==4.12.2
 certifi==2022.12.7
 charset-normalizer==3.0.1
+colorama==0.4.6
 docutils==0.19
+furo==2023.5.20
 idna==3.4
 imagesize==1.4.1
 importlib-metadata==6.0.0
 Jinja2==3.1.2
+livereload==2.6.3
 MarkupSafe==2.1.2
 packaging==23.0
 Pygments==2.14.0
 pytz==2022.7.1
 requests==2.28.2
+six==1.16.0
 snowballstemmer==2.2.0
-sphinxcontrib-applehelp==1.0.3
+soupsieve==2.4.1
+Sphinx==6.1.3
+sphinx-autobuild==2021.3.14
```

Comment:
I added this too, since it makes `make livehtml` work, which is useful :-)

Reply:
FWIW, I was previously manually sorting these, so that the explicit list is at the top, and the implied list is at the bottom (with the goal being making it easier for someone to upgrade later without accidentally dropping something)

Reply:
Do you think it's important at this stage? No strong opinion, but having to manually sort it rather than doing `pip freeze > requirements.txt` is not my favorite :-P

Reply:
I think it is otherwise unclear to the next person who needs to update the file to get new versions. There should not be much sorting, just a list at the top of which things are being explicitly installed.

Reply:
Updated, PTAL!